swagger.json

```diff
@@ -870,6 +870,50 @@
                 ]
             }
         },
+        "/event/{cdbid}/audience": {
+            "put": {
+                "summary": "Set the target audience for an event.",
+                "description": "Authentication is required.",
+                "operationId": "setEventAudience",
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/cdbid"
+                    },
+                    {
+                        "name": "body",
+                        "in": "body",
+                        "description": "An object containing the audience information to set for the event.",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/EventAudienceInfo"
+                        }
+                    }
+                ],
+                "responses": {
+                    "202": {
+                        "$ref": "#/responses/CommandQueued"
+                    },
+                    "401": {
+                        "$ref": "#/responses/Unauthorized"
+                    },
+                    "403": {
+                        "$ref": "#/responses/Forbidden"
+                    },
+                    "400": {
+                        "$ref": "#/responses/APIProblem"
+                    },
+                    "404": {
+                        "description": "Offer or label not found."
```

Comment:
@cyberwolf should this be "Event not found." ? Audience is only supported on event.